test/chargebeex/subscription_test.exs

```diff
@@ -8,6 +8,257 @@ defmodule Chargebeex.SubscriptionTest do
 
   setup :verify_on_exit!
 
+  describe "retrieve" do
+    test "with bad authentication should fail" do
+      unauthorized = Common.unauthorized()
+
+      expect(
+        Chargebeex.HTTPClientMock,
+        :get,
+        fn url, body, headers ->
+          assert url == "https://test-namespace.chargebee.com/api/v2/subscriptions/1234"
```

Comment:
Move to module variable `@url` ?

Reply:
We are testing different endpoints in this module:

- `/v2/subscriptions`;
- `v2/subscriptions/foobar`;
- `/v2/subscriptions/foobar/delete`;
- `/v2/customers/foobar/subscription_for_items`;
- `/v2/subscriptions/foobar/update_for_items`.

Should we create one module attribute for each?
If we decide to use module attributes for these values I guess we should also refactor the other test modules in the project in order to have the same pattern, WDYT?

Reply:
Yes

Reply:
Since this would impact multiple test modules, IMO this refactoring is outside the scope of this PR. I think it should be done in a separate PR, WDYT @klacointe ?